Sources/MapboxMaps/Foundation/CredentialsManager.swift

```diff
@@ -0,0 +1,100 @@
+import Foundation
+
+/// Convenience class that holds MapboxMaps related secrets.
+///
+/// It's possible to create `CredentialsManager` instances as you need them,
+/// however it's convenient to use the default object (`default`).
+///
+/// For example, we recommend that the Mapbox access token be set in
+/// `application(_:didFinishLaunchingWithOptions:)` rather than relying on the
+/// value in your application's Info.plist:
+///
+///     ```
+///     func application(_ application: UIApplication, didFinishLaunchingWithOptions launchOptions: [UIApplication.LaunchOptionsKey: Any]?) -> Bool {
+///         // Override point for customization after application launch.
+///         CredentialsManager.default.accessToken = "overridden-access-token"
+///         return true
+///     }
+///     ```
+public class CredentialsManager {
+
+    /// Access token
+    public var accessToken: String
```

Comment:
Why not make this optional and avoid the need to treat the empty string as special?

Reply:
Did you intend for the setter to be public?